doc/dev/developer_guide/dash-devel.rst

```diff
@@ -1026,6 +1026,59 @@ Alerts and notifications
 Default notification should have `text-info` color. Success notification should
 have `text-success` color. Failure notification should have `text-danger` color.
 
+Accessibility
+.............
+
+We follow the `W3C WCAG2.1 <https://www.w3.org/TR/WCAG21/>`_  level A accessibility conformance guidelines. 
+By implementing accessibility best practices, you are also improving the usability of the site for all users.
```

Comment:
Suggest "By following accessibility best practices we improve the usability of the site for all users"